### PR TITLE
New package: the-way-0.14.4.

### PR DESCRIPTION
**Pull request by jgarte**
Home page: https://github.com/out-of-cheese-error/the-way

General

    [x] This is a new package and it conforms to the quality requirements

Have the results of the proposed changes been tested?

    [x] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me

Does it build and run successfully?

    [x] I built this PR locally for my native architecture, (x86_64)

### CODE REVIEW

File: srcpkgs/the-way/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'the-way'
+pkgname=the-way
+version=0.14.4
+revision=1
+archs="i686 x86_64"
```
Comment (paper42) [inline srcpkgs/the-way/template:5]: could you explain this?

Comment (jgarte) [reply srcpkgs/the-way/template:5]: I think that was from the xnew template I used.

Maybe we should change xnew upstream to not include that line by default?

Comment (jgarte) [reply srcpkgs/the-way/template:5]: > could you explain this?

I'm removing it.

File: srcpkgs/the-way/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+pkgname=the-way
+version=0.14.4
+revision=1
+build_style=cargo
+short_desc="Code snippets manager for your terminal"
+maintainer="jgart <jgart@dismail.de>"
+license="MIT"
+homepage="https://github.com/out-of-cheese-error/the-way"
+distfiles="https://github.com/out-of-cheese-error/${pkgname}/archive/refs/tags/v${version}.tar.gz"
+checksum=@7ffce9b744ccbc9cdf8c7968f50ce8b580d66d9d40c477d090b6a45aa5560ccd
```
Comment (paper42) [inline srcpkgs/the-way/template:11]: why are you using content checksum?

File: srcpkgs/the-way/template [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'the-way'
+pkgname=the-way
+version=0.14.4
+revision=1
+build_style=cargo
+short_desc="Code snippets manager for your terminal"
+maintainer="jgart <jgart@dismail.de>"
+license="MIT"
+homepage="https://github.com/out-of-cheese-error/the-way"
+distfiles="https://github.com/out-of-cheese-error/${pkgname}/archive/refs/tags/v${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/the-way/template:10]:
```suggestion
distfiles="https://github.com/out-of-cheese-error/the-way/archive/refs/tags/v${version}.tar.gz"
```

File: srcpkgs/the-way/template (hunk 2 above)
Comment (jgarte) [reply srcpkgs/the-way/template:11]: Not sure, what should I be using instead?

File: srcpkgs/the-way/template (hunk 3 above)
Comment (jgarte) [reply srcpkgs/the-way/template]: I force pushed the changes so resolving this.

File: srcpkgs/the-way/template (hunk 2 above)
Comment (jgarte) [reply srcpkgs/the-way/template]: I just pushed the non content checksum so I'm resolving this conversation.

**Pull request closed without merging**